dependency/Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: retrieve
```

Comment:
I noticed there are no dependency tests, which I would hope for if we're compiling ourselves. Is there a reason you omitted them?